data/tcp-listener.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @title TCP Listener: Ping
+ * @difficulty beginner
```

Comment:
Because http server examples are makred `intermediate` (ref: https://examples.deno.land/http-server), I think this also should be labeled `intermediate`

Reply:
Switched this up, I suppose my understanding of the user base is a bit off.